organisations/organisations.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package organisations;
+
+option go_package = "grpc/organisationspb";
+
+import "google/protobuf/empty.proto";
+
+message Organisation {
+    string id = 1;
+
+    string name = 2;
+
+    Member creator = 3;
+    
+    repeated Member members = 4;
+}
+
+message Member {
+    string id = 1;
+}
+
+service OrganisationService {
+    rpc CreateOrganisation(CreateOrganisationRequest) returns (google.protobuf.Empty) {}
+
+    rpc DeleteOrganisation(OrganisationFilterRequest) returns (google.protobuf.Empty) {}
+
+    rpc AddMembersToOrganisation(AddMembersToOrganisationRequest) returns (google.protobuf.Empty) {}
+
+    rpc RemoveMembersFromOrganisation(DeleteMembersFromOrganisationRequest) returns (google.protobuf.Empty) {}
```

Comment:
Maybe it's better to have a `UpdateGroup` instead of having an RPC for every use case?